templates/core/terraform/variables.tf

```diff
@@ -60,8 +60,8 @@ variable "terraform_state_container_name" {
 }
 
 variable "resource_processor_number_processes_per_instance" {
-  type        = string
-  default     = "2"
+  type        = number
+  default     = 5
```

Comment:
Why 5? :)  Seems odd not to have power of 2

Reply:
no particular reason - and given the 'host' process also exists this would make the number of python processes an even number